articles/create-login-and-register-system-with-C#-winforms/index.md

```diff
@@ -0,0 +1,430 @@
+### Introduction
+
+C# or C-sharp is an object-oriented programming language developed by Microsoft that runs on a framework called .Net framework. The language has been used to create many applications, including web applications, mobile applications, computer games, desktop applications, database applications, virtual reality applications, etc.
+
+### Prerequisites
+
+* A code editor a good example is [Visual Studio](https://visualstudio.microsoft.com/vs/).
+* Knowledge of C#, SQL.
+* Running relational database management in this case, we will use [MySQL](https://www.mysql.com/).
+* Visual Studio Knowledge.
+
+### Step I: Create a database and table with required columns.
+
+``` sql
+
+Create database userdata;
+```
+
+![DATABASE CREATION]((/engineering-education/create-login-and-register-system-with-C#-winforms/create-db-cmd.png))
+
+Create table command
+
+``` sql
+
+CREATE TABLE `user_info` ( `id` INT NOT NULL AUTO_INCREMENT , `names` VARCHAR(200) NOT NULL , `username` VARCHAR(100) NOT NULL , `password` VARCHAR(255) NOT NULL , PRIMARY KEY (`id`)) ;
+```
```

Comment:
Once again, you've defined larger sizes, just make sure you read more about it. Just an FYI.